test/nn/models/test_schnet.py

```diff
@@ -0,0 +1,63 @@
+import torch
+
+from torch_geometric import seed_everything
+from torch_geometric.data import Batch, Data
+from torch_geometric.nn import SchNet
+from torch_geometric.testing import withPackage
+from torch_geometric.transforms import RadiusGraph
+
+
+def generate_data():
+    return Data(
+        z=torch.randint(1, 10, (20, )),
+        pos=torch.randn(20, 3),
+        y=torch.tensor([1.]),
+    )
+
+
+@withPackage('torch_cluster')
+@withPackage('ase')
+def test_schnet():
+    seed_everything(0)
```

Comment:
Can we drop the `seed_everything` calls here? I don't think they are necessary.

Reply:
sure, I did this out of habit.  I've seen pathological cases where a test may fail for a given PRNG state. Arguably overly paranoid though to do this everywhere.